lib/shopify-cli/messages/messages.rb

```diff
@@ -237,6 +237,8 @@ module Messages
           },
           environment_header: "{{bold:Environment}}",
           env: "%-17s = %s",
+          identity_header: "{{bold:Identity}}",
+          identity_is_shopifolk: "{{v}} you are a shopifolk!",
```

Comment:
I'd suggest just a smidge more context:

```suggestion
          identity_is_shopifolk: "{{v}} Checked user settings: you’re Shopify staff!",
```